app/webpacker/scripts/copy-tracking.js

```diff
@@ -1,29 +1,22 @@
-function CopyTracking($module) {
-  this.$module = $module;
-}
+import { triggerAnalyticsEvent } from './analytics'
 
-const triggerAnalyticsEvent = (category, action) => {
-  ga("send", "event", {
-    eventCategory: category,
-    eventAction: action,
-    transport: "beacon",
-    nonInteraction: true
-  });
-};
+function CopyTracking ($module) {
+  this.$module = $module
+}
 
-CopyTracking.prototype.init = function() {
-  const $module = this.$module;
+CopyTracking.prototype.init = function () {
+  const $module = this.$module
 
   if (!$module) {
-    return;
+    return
   }
 
-  $module.addEventListener("copy", e => {
-    if (event.clipboardData) {
```

Comment:
Standard Identified that event was not declared. In this case we should have used `e` which is the passed in value.